code/controllers/subsystem/starmap.dm

```diff
@@ -135,11 +160,16 @@ SUBSYSTEM_DEF(starmap)
 			current_planet = current_system.planets[1]
 		else if(in_transit_planet)
 			current_planet = to_planet
+			current_planet.event.init_event()
```

Comment:
Unless you don't want events when jumping systems, this shouldn't be in this conditional block